src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1018,16 +995,15 @@
   (let [csr-path (path-to-cert-request csrdir subject)]
     (if (fs/exists? csr-path)
       (if (fs/delete csr-path)
-        (let [msg (str "Deleted certificate request for " subject)]
```

Comment:
looks like these few msg end up being logged, not returned over an http request or something; i think that makes these `trs` instead of `tru`?